README.md

```diff
@@ -110,12 +93,10 @@ Image Hub supports Envoy-based data planes. It is compatible with and can be dep
 <p style="clear:both;">
 <h2><a href="https://github.com/layer5io/learn-layer5">Learn Layer5</a></h2>
 <a href="https://github.com/layer5io/learn-layer5">
-<picture>
-<source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/99991521/182012730-19bf6776-4ebd-460f-b6bb-75105f2a10bc.svg#gh-dark-mode-only">
-  <img src=".github/assets/images/layer5/5-light-bg.png#gh-light-mode-only"
-style="margin:10px;" width="125px"
-alt="Layer5" align="left" />
-</picture></a>
+  <img src=".github/assets/images/layer5/5-light-small.svg"
+style="float:left;margin:10px;" width="100px"
+alt="Learn Layer5" align="left" />
+</a>
 <a href="https://github.com/layer5io/learn-layer5">Learn Layer5</a> is a sample application for learning how service meshes work which also provides <a href="https://layer5.io/smi">SMI Conformance testing</a>.
```

Comment:
Add &nbsp  &nbsp &nbsp infront for better alignment.

Reply:
Added 👍